packages/core/src/shared/lsp/utils/platform.ts

```diff
@@ -89,8 +103,7 @@ export function createServerOptions({
         if (isDebugInstance()) {
             args.unshift('--inspect=6080')
         }
-        const lspProcess = new ChildProcess(executable, args)
-
+        const lspProcess = createNodeProcess(executable, args)
```

Comment:
`createServerOptions` also shouldn't need changes. instead, the *caller* to `createServerOptions` should have already decided the right `nodePath` (which is called `executable`) here, after checking `validateNodeExe`